src/styles.scss

```diff
@@ -6,3 +6,31 @@ body{
     background-color: rgb(239, 216, 202);
 
 }
+
+input {
```

Comment:
please give on element input
class name and use in scss
its more common

Reply:
fixed